components/director/internal/graphql/scalars/timestamp.go

```diff
@@ -0,0 +1,49 @@
+package scalars
+
+import (
+	"encoding/json"
+	"io"
+	"log"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+type Timestamp time.Time
+
+func (y *Timestamp) UnmarshalGQL(v interface{}) error {
+
+	if v == nil {
+		return nil
+	}
+
+	tmpStr, ok := v.(string)
```

Comment:
Is there any purpose you do not use convertToString here?

Reply:
no, I just missed it! I've changed it